### PR TITLE
chore: convert 4 in 'developer/16/context/...'

### PR DESCRIPTION
**Pull request by Meng-Heng**

### CODE REVIEW

File: developer/16.0/context/model-editor.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+## Lexical model component files
+
+A lexical model can have multiple source files, all of which can be
+managed in this one editor. The source files currently supported by the
+editor are:
+
+| File type         | Tabs            | Description                                                                                                             |
+|-------------------|-----------------|-------------------------------------------------------------------------------------------------------------------------|
+| `file``.model.ts` | Details, Source | The primary lexical model source file, a TypeScript module.                                                             |
+| `wordlist``.tsv`  | wordlist.tsv    | One or more wordlists, UTF-8 format tab-separated value files. These files are referenced in the .model.ts source file. |
```
Comment (darcywong00) [inline developer/16.0/context/model-editor.md:17]: I think we can remove the extra backticks

```suggestion
| `file.model.ts` | Details, Source | The primary lexical model source file, a TypeScript module.                                                             |
| `wordlist.tsv`  | wordlist.tsv    | One or more wordlists, UTF-8 format tab-separated value files. These files are referenced in the .model.ts source file. |
```

**Review by Meng-Heng — COMMENTED** [review]
Double backticks are removed.

File: developer/16.0/context/model-editor.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+models for predictive text.
+
+## Lexical model component files
+
+A lexical model can have multiple source files, all of which can be
+managed in this one editor. The source files currently supported by the
+editor are:
+
+| File type         | Tabs            | Description                                                                                                             |
+|-------------------|-----------------|-------------------------------------------------------------------------------------------------------------------------|
```
Comment (Meng-Heng) [inline developer/16.0/context/model-editor.md:15]:
```suggestion
|-------------------|-----------------|-------------------|

```

File: developer/16.0/context/model-editor.md [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+The lexical model editor allows you to create and maintain lexical
+models for predictive text.
+
+## Lexical model component files
+
+A lexical model can have multiple source files, all of which can be
+managed in this one editor. The source files currently supported by the
+editor are:
+
+| File type         | Tabs            | Description                                                                                                             |
```
Comment (Meng-Heng) [inline developer/16.0/context/model-editor.md:14]:
```suggestion
| File type         | Tabs            | Description          |
```

File: developer/16.0/context/model-editor.md [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+
+## Lexical model component files
+
+A lexical model can have multiple source files, all of which can be
+managed in this one editor. The source files currently supported by the
+editor are:
+
+| File type         | Tabs            | Description          |
+|-------------------|-----------------|-------------------|
+
```
Comment (Meng-Heng) [inline developer/16.0/context/model-editor.md:16]:
```suggestion
```

**Review by darcywong00 — APPROVED** [review]
lgtm

**Comment by darcywong00** [comment]
I changed the base to master and will merge this.

**Pull request merged**